CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ Last releases: [[1.9.0] - 2019-05-22](#190---2019-05-22) and [[1.8.0] - 2019-04-
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [Unreleased]
+
+### Changed
+
+- `eqVneq` lemma is changed from `{x = y} + {x != y}` to
+  `eq_xor_neq x y (y == x) (x == y)` which allows to use it as
+   a view like `eqP`, in addition providing simultaneous destruction
```

Comment:
I would remove the comparison with `eqP` which is less relevant than the comparison to `eqVneq`.

Reply:
done